.gitignore

```diff
@@ -1 +1,232 @@
-/node_modules
\ No newline at end of file
+/node_modules# Logs
```

Comment:
I botched my commit message where I tried to explain this, but I created this .gitignore using `npx gitignore node`.

Reply:
I had no idea that was a thing. How handy!